### ISSUE TITLE
Question - Web Player renderer config?

### ISSUE THREAD

**Issue by threedguru** [UniversalMediaServer/UniversalMediaServer#3766]
Hi,

I asked about this in the forum:

https://www.universalmediaserver.com/forum/viewtopic.php?t=15557#p46766

The web browser(s) in VR headsets support 3D formats. It would be very nice if I can tell the Web Player to enable 2D to 3D transformation - as then users could get away without a media player and it would potentially work better than a DLNA media player. I cannot find a config file for the Web Player. I can see the Transcode folder in the Web Player, so this should just be a question of turning the transformation on in in the right place. Where is best to do that?

I have struggled to find the perfect VR media player for UMS. Forward / rewind is a problem with DLNA in general. I have mitigated that by advising the enabling of Chapters in UMS in the guide, which works really well in BigScreen for example. However, Pigasus won't play the chapters or resume, for example and Sky Box has stopped being able to handle the Transcode folder. BigScreen is almost perfect except it assumes everything is in 16:9 format (stretches 4:3) for example. I was about to address this by implementing Letter Box and Zoom options in the 2D to 3D script. Your thoughts on that would be appreciated.

Supporting 2D to 3D in the Web Player would potentially address a number of issues. If it doesn't use DLNA for Transcoded videos (why would it?) then rewind and forward should work better. It also means users don't have to install a specialised media player. I am also curious how aspect ratio is handled. If I put Letter Box and Zoom support in the script (which is trivial to implement) then that would support the browser based solutions too.

I will be proving this out in the Oculus Browser and Firefox Reality Browser (just noticed it has now branched to become Wolvic browser) in my Meta Quest 2 and Oculus Quest 1 headsets.

Wolvic:
https://www.wolvic.com/en/

As an FYI, the latest 2D to 3D algorithm works amazingly well, see here. This versio of the algorithm is already in the UMS main branch:

https://iwantaholodeck.com/algorithmic-tuning-motion-and-alignment/

**Comment by SubJunk** [comment]
Yeah it would be great to have browser configs since they are hardcoded now
